src/cheat_sheet.py

```diff
@@ -56,17 +47,36 @@ def close(self):
         self.canvas.unregister("mouse", self.mouse)
         self.canvas.close()
 
+    def debounce_resize(self, width: int, height: int):
```

Comment:
Presumably this is because draw gets called multiple times in quick succession?  Might be worth a comment to say why

Reply:
I don't know why. Draw just get called multiple times for no reason what I can tell.